client/components/wizard-progress-bar/README.md

```diff
@@ -0,0 +1,70 @@
+Wizard Progress Bar
+===
+
+A Progress Bar for use with a custom wizard. It has buttons to go backward and forward through the steps of the wizard and will display the progress. This Component is unrelated to the `Wizard` Component.
+
+## Usage
+
+```jsx
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import WizardProgressBar from 'components/wizard-progress-bar';
+
+export default class WizardProgressBarExample extends Component {
+
+	static defaultProps = {
+		numberOfSteps: PropTypes.number.isRequired,
+	};
+
+	state = {
+		currentStep: 1
+	}
+
+	handleNextButtonClick = () => {
+		this.setState( {
+			currentStep: Math.min( this.props.numberOfSteps, this.state.currentStep + 1 ),
+		} );
+	};
+
+	handlePreviousButtonClick = () => {
+		this.setState( {
+			currentStep: Math.max( 1, this.state.currentStep - 1 ),
+		} );
+	};
+
+	render() {
+		return (
+			<div>
+				<WizardProgressBar
+					currentStep={ this.state.currentStep }
+					nextButtonClick={ this.handleNextButtonClick }
+					numberOfSteps={ this.props.numberOfSteps }
+					previousButtonClick={ this.handlePreviousButtonClick }
+				/>
+			</div>
+		);
+	}
+}
+```
+
+### Props
+
+Props are displayed as a table with Name, Type, Default, and Description as headings.
```

Comment:
We should remove this line.

Reply:
Ugh, that was sloppy to leave that in there 😬